.php-cs-fixer.dist.php

```diff
@@ -1,18 +1,22 @@
 <?php
 
 /*
- * This file is part of Monsieur Biz's  for Sylius.
+ * This file is part of Monsieur Biz' Sylius Plus Adapter plugin for Sylius.
+ *
  * (c) Monsieur Biz <sylius@monsieurbiz.com>
- * For the full copyright and license information, please view the LICENSE
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
  * file that was distributed with this source code.
  */
 
 declare(strict_types=1);
 
 $header = <<<'HEADER'
-This file is part of Monsieur Biz's  for Sylius.
+This file is part of Monsieur Biz' Sylius Plus Adapter plugin for Sylius.
+
 (c) Monsieur Biz <sylius@monsieurbiz.com>
-For the full copyright and license information, please view the LICENSE
+
+For the full copyright and license information, please view the LICENSE.txt
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** why add `.txt` the file has no extension?‏

Reply:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** we need to modify the maker plugin to add it automatically